src/libraries/System.Runtime/tests/System/UInt64Tests.cs

```diff
@@ -120,6 +120,7 @@ public static IEnumerable<object[]> ToString_TestData()
             yield return new object[] { (ulong)123, "E", customFormat, "1~230000E&002" };
             yield return new object[] { (ulong)123, "F", customFormat, "123~00" };
             yield return new object[] { (ulong)123, "P", customFormat, "12,300.00000 @" };
+            yield return new object[] { ulong.MaxValue, "n5", null, ulong.MaxValue.ToString("n5") };
```

Comment:
Can you use `customFormat` defined above to avoid dependency on machine locale?

The test is a lot less useful if it is just going to test that `ToString("n5")` returns the same value everytime. We want to test the value that it returns.